lib/win-exception-handler/build.rs

```diff
@@ -1,10 +1,10 @@
 fn main() {
-    #[cfg(target_os = "windows")]
-    {
-        use cmake::Config;
-        let project_name = "exception_handling";
-        let dst = Config::new(project_name).build();
-        println!("cargo:rustc-link-search=native={}", dst.display());
-        println!("cargo:rustc-link-lib=static={}", project_name);
```

Comment:
Why are these lines deleted?  Surely, we still need to link the compiled output, or does `cc` somehow do that as well?

Reply:
Sorry, yes, cc automatically

1. Determines the compiler to use based on the environment
2. Specifies additional linker search directories
3. Additional libraries to link to